1.7/usage/tutorials/jenkins/index.md

```diff
@@ -66,13 +69,14 @@ as NFS, CIFS, and so on.
 
 If you only want to run Jenkins in a development environment, it's trivial
 to pin it to a single agent in the DC/OS cluster. Create the file
-`options.json` with the configuration below:
+`options.json` with the configuration below, modifying
+`storage.pinned-hostname` to correspond with an agent IP in your DC/OS cluster:
 
 ```bash
 $ cat options.json
 {
     "jenkins": {
-        "pinned-hostname": "10.100.100.88"
+        "storage.pinned-hostname": "10.100.100.88"
```

Comment:
See https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/J/jenkins/11/config.json for reference.